WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -277,8 +277,8 @@ private boolean editPostActivityIsNoLongerLoadingImages() {
         return editPostActivity.getAztecImageLoader().getNumberOfImagesBeingDownloaded() == 0;
     }
 
-    private void setNightModeAndWait(boolean isNightMode) {
-        setNightMode(isNightMode);
+    private void setLightModeAndWait() {
```

Comment:
I wonder if it's still useful to call `setLightModeAndWait()` before every single call to `takeScreenshot` if we're now only taking light mode screenshots for every screen and never dark modes?

Like, we could instead just switch to light mode at the very beginning of the test, by calling this method only once at the start, and not having to call it again before every individual `takeScreenshot` calls.

The purpose of this method back when it took a boolean was to be able to navigate to a screen, switch to night mode to take one screenshot, then switch to light mode to take another screenshot of the same screen, then continue navigating to the next screen, switch, screenshot, switch, screenshot, continue, etc. If we don't constantly switch mode anymore we could then avoid the `idleFor(5000)` that would happen before each screenshot, and speed up the tests there.

That is all assuming that the `idleFor(5000)` here didn't serve a double purpose of not only waiting for the mode to change from light to dark in case it took some time, but also to wait for the screen content (regardless of if light/night mode needed switching) finished loading? But if that;s case the case I'm not sure why we grouped those two lines together in a single function if there're not _both_ related to mode switching 🙃

Reply:
@AliSoftware Yup I agree, what was I thinking 😅 . 

>Like, we could instead just switch to light mode at the very beginning of the test, by calling this method only once at the start, and not having to call it again before every individual take screenshot calls.

yup 💯 . I will make that change and push. 


>That is all assuming that the idleFor(5000) here didn't serve a double purpose of not only waiting for the mode to change from light to dark in case it took some time, but also to wait for the screen content (regardless of if light/night mode needed switching)

IdleFor(5000) was only used for waiting for the mode to change. So it's safe to do the change you suggested.